pkg/utils/annotation.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+const (
+	// DeployModeAnnotationKey describe the klusterlet deploy mode when importing a managed cluster.
+	DeployModeAnnotationKey string = "import.open-cluster-management.io/klusterlet-deploy-mode"
+	DeployModeValueHosted          = "Hosted"
+
+	// DeployAddonHostingKey
```

Comment:
add some comments?

Reply:
Yes, should I add a link to `addon-framework` to describe this key, and I saw `constant` pkg have this key, I don't whether it is ok to use it directly: https://github.com/open-cluster-management-io/addon-framework/blob/12c9395bde82925b8d238e300a0d9cae855f91d8/pkg/addonmanager/constants/constants.go#L46

Reply:
Fixed.